tools/cpp-common/gmm_eigen.h

```diff
@@ -1,9 +1,12 @@
 #include <cmath>
```

Comment:
Isn't this file supposed to be moved to /src/cpp/shared? We should delete it from here, then. Need to check that all the references in the codebase are corrected before doing that, though.

Reply:
I prefer not to delete gmm_eigen.h until eigen vectorized will be implemented.